FAQ.md

```diff
@@ -893,3 +896,23 @@ moving the file/folder in the last snapshot, too.
 
 1. remove the overlast snapshot (the one where you moved the folder manually)
    to avoid problems with permissions when you try to restore from that snapshot
+
+
+# Testing & Building
+
+## SSH related tests are skipped
+Some of the tests need an available SSH server.
+They get skipped if this is not the case.
+Plese see
+[Setup SSH Server to run unit tests](#setup-ssh-server-to-run-unit-tests).
+
+## Setup SSH Server to run unit tests
+The goal is to log into the SSH server via `ssh localhost` without using
+a password.
+- Generate an RSA keypair executing `ssh-keygen`.
```

Comment:
I think it is also required for the unit test that the key is not password-protected.
Not sure if this adds security risks requiring more config changes...

Reply:
I added this point to the FAQ.